DrinkManagerWeb/Models/ViewModels/UserViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace DrinkManagerWeb.Models.ViewModels
+{
+    public class UserViewModel
+    {
+        public string Id { get; set; }
+
+        [Required]
+        [EmailAddress]
+        [Display(Name = "Email")]
+        public string Email { get; set; }
+
+        [DataType(DataType.Password)]
+        [Required]
+        [RegularExpression("^[a-zA-ZżźćńółęąśŻŹĆĄŚĘŁÓŃ]+\\d+$", ErrorMessage = "Use letters (lowercase and/or uppercase) and at least one digit.")]
```

Comment:
I think that password validation should be handled with default identity rules in startup, it would also allow us to check for duplicate email.

`            services.AddDefaultIdentity<AppUser>(options =>
                {
                    options.SignIn.RequireConfirmedAccount = false;
                    options.Password.RequireUppercase = false;
                    options.Password.RequireNonAlphanumeric = false;
                    options.User.RequireUniqueEmail = true;
                    options.Password.RequiredLength = 6;
                    options.Password.RequireDigit = true;
                })
                .AddRoles<IdentityRole>()
                .AddEntityFrameworkStores<DrinkAppContext>();`

Reply:
Done.